bpf/tracing.h

```diff
@@ -94,15 +119,15 @@ static __always_inline u8 correlated_request_with_current(tp_info_pid_t *existin
         return 0;
     }
 
-    u64 pid_tid = bpf_get_current_pid_tgid();
+    //u64 pid_tid = bpf_get_current_pid_tgid();
```

Comment:
just noting those here in case they were not intended

Reply:
Yeah I removed the pid checking, I'll clean up this code in the next round of work I'm doing. I moved the type to be in the key, so now we don't have to worry about the pid.